app/sections/root/root.component.html

```diff
@@ -1,5 +1 @@
-<h1>
-  Main page
-</h1>
-
-
+<posts></posts>
```

Comment:
it would be nice to have some `limit` parameter here, probably service will fetch all posts, and if we want to have it more flexible, it would be nice to have possibility to change the posts' quantity . what do you think ?